docs/Microsoft.ReactNative.xml

```diff
@@ -0,0 +1,495 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doc>

```

Comment:
>doc [](start = 1, length = 3)

Do we need to add this file to the docs?

Reply:
This is for IntelliSense (xml can be dropped next to winmd to produce intellisense on winrt apis)